std/json.d

```diff
@@ -166,7 +165,8 @@ struct JSONValue
     }
 
     /// Value getter/setter for $(D JSON_TYPE.STRING).
-    /// Throws $(D JSONException) for read access if $(D type) is not $(D JSON_TYPE.STRING).
+    /// Throws $(D JSONException) for read access if $(D type) is not
```

Comment:
Yeah, that's what I meant.  Just a consistency change, functionally it was fine.